docs/ecctl-release-notes.asciidoc

```diff
@@ -3,8 +3,60 @@
 
 This section summarizes the changes in each {n} release.
 
+* <<{p}-release-notes-v1.0.0-beta2>>
 * <<{p}-release-notes-v1.0.0-beta1>>
 
+[id="{p}-release-notes-v1.0.0-beta2"]
+=== Elastic Cloud Control (ecctl) v1.0.0-beta2
+++++
+<titleabbrev>v1.0.0-beta2</titleabbrev>
+++++
+
+<<{p}-release-notes-v1.0.0-beta2-whats-new,What's new>> | <<{p}-release-notes-v1.0.0-beta2-bug-fixes,Bug fixes>> | <<{p}-release-notes-v1.0.0-beta2-breaking-changes,Breaking changes>> | <<{p}-release-notes-v1.0.0-beta2-changelog,Changelog>>
+
+Welcome to the v1.0.0-beta2 release of {n}! This version brings new features, some breaking changes, and bug fixes. 
+
+Download the release binaries:
+
+[%hardbreaks]
+https://download.elastic.co/downloads/ecctl/v1.0.0-beta2/ecctl_v1.0.0-beta2_darwin_amd64.tar.gz[ecctl_v1.0.0-beta2_darwin_amd64.tar.gz]
```

Comment:
Since this PR change: https://github.com/elastic/ecctl/pull/152/files#diff-0c48475a77204c9aa7b76e994cf24a6eR11 the links no longer use the prefixed `v` in the link. so all the `v`s need to be removed.

Reply:
I pushed a couple of commits to remove all the occurrences of `v` in the links and descriptions.